src/asInput/index.jsx

```diff
@@ -89,15 +101,17 @@ const asInput = (WrappedComponent, labelFirst = true) => {
     render() {
       const { description, error, describedBy } = this.getDescriptions();
       return (
-        <div className={styles['form-group']}>
+        <div className={classNames(styles['form-group'])}>
```

Comment:
Any harm in adding classNames?

Reply:
I don't think there is any harm, but why do you think it is useful here if there are no class names to join?

Reply:
Ah, my bad. I had added class names and then removed them. I'll switch it back.